assemblies/assembly-ba-artificial-intelligence.adoc

```diff
@@ -0,0 +1,56 @@
+[id='assembly-ba-artificial-intelligence']
+= Integrating artificial intelligence with {PRODUCT}
+ifdef::context[:parent-context: {context}]
+
+:context: artificial-intelligence
+
+= What is artificial intelligence?
+When you think about artificial intelligence (AI), machine learning and big data might come to mind. But machine learning is only part of the picture. Artificial intelligence encompasses the following technologies:
```

Comment:
I am not sure the bullet points below cover all of the areas in AI, so I suggest you change the wording on this paragraph to make it clear these are some of the technologies, not all.

Reply:
Thanks, Edson. Changed emcompasses to includes.